openstack/objectstorage/v1/containers/results.go

```diff
@@ -104,16 +105,19 @@ type GetHeader struct {
 	StoragePolicy    string    `json:"X-Storage-Policy"`
 	TempURLKey       string    `json:"X-Container-Meta-Temp-URL-Key"`
 	TempURLKey2      string    `json:"X-Container-Meta-Temp-URL-Key-2"`
+	Timestamp        float64   `json:"-"`
```

Comment:
@jtopjian you can use `ContentLength  int64     `json:"Content-Length,string"``, see for example https://github.com/gophercloud/gophercloud/blob/c9471fccb2bb90d41832195b1134ac1c6f50f2bb/openstack/objectstorage/v1/objects/results.go#L136

Reply:
Oh nice - thanks!